src/driver.rs

```diff
@@ -39,13 +36,14 @@ pub fn driver() -> io::Result<bool> {
         .collect();
 
     /* convert to runner env variable */
-    let host = host.replace('-', "_").replace('.', "_").to_uppercase();
-    let runner = format!("CARGO_TARGET_{}_RUNNER", host);
+    let host = host.replace(['-', '.'], "_").to_uppercase();
+    let runner = format!("CARGO_TARGET_{host}_RUNNER");
```

Comment:
```suggestion
    let runner = format!("CARGO_TARGET_{}_RUNNER", host);
```
Could you undo your change here to keep the project building on older Rust versions? Then it should be fine to merge (even with a bunch of clippy warnings, that are not coming from your change).